packages/ncrack/build.sh

```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://nmap.org/ncrack/
+TERMUX_PKG_DESCRIPTION="high-speed network authentication cracking tool"
+TERMUX_PKG_VERSION=0.5.0
+TERMUX_PKG_SHA256=4dd4abfa7c38edafd0da3a3f2ad76f9fae476e147042def87d027879444bd0b9
+TERMUX_PKG_SRCURL=https://github.com/nmap/ncrack/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_BUILD_IN_SRC=yes
+TERMUX_PKG_DEPENDS="openssl, openssh, openssl-dev"
+
+termux_step_post_configure () {
+        echo
+        echo
+        
+        cd /opensshlib/
```

Comment:
The problem is this line not line 18

Reply:
So what should I do?

Reply:
The code has to be executed before `make` and after `configure`